server/src/models/Podcast.js

```diff
@@ -45,6 +46,18 @@ class Podcast extends Model {
           from: "podcasts.userId",
           to: "users.id"
         }
+      },
+      votes: {
+        relation: Model.ManyToManyRelation,
+        modelClass: Vote,
+        join: {
+          from: 'podcasts.id',
+          through: {
+            from: 'votes.podcastId',
+            to: 'votes.userId'
+          },
+          to: 'users.id'
+        }
```

Comment:
Unfortunately, this relation is no beuno for a few reasons, and should be removed. 

- you never use this functionality, there is no `podcast.$relatedQuery("votes")` (which is good, I like your votesRouter implementation actually) 
- the relation itself is wrong. The relationMapoing designates that you are trying to retrieve a modelClass of `Vote` and have dubbed the relation `votes`, but the join is representative of a many to many from podcasts to users _through_ votes
- even if you changed this to a many to many between users and podcasts, why would we want to know which users have voted for a podcast? if there was a feature that demanded this, then I could see the value

Reply:
additionally, I see no relation between the key players in your feature: podcasts and votes. You don't need to add it based on your current functionality, but that would have been far preferable